app/models/user.rb

```diff
@@ -762,7 +762,14 @@ def refresh_counts!
           post_count: Post.for_user(id).count,
           post_deleted_count: Post.for_user(id).deleted.count,
           post_update_count: PostVersion.for_user(id).count,
-          note_count: NoteVersion.where(updater_id: id).count
+          favorite_count: Favorite.for_user(id).count,
+          wiki_edit_count: WikiPageVersion.for_user(id).count,
+          note_count: NoteVersion.for_user(id).count,
+          pool_edit_count: PoolVersion.for_user(id).count,
+          artist_edit_count: ArtistVersion.for_user(id).count,
+          own_post_replaced_count: PostReplacement.for_user(id).count,
```

Comment:
These two are not correct. It should count posts that were uploaded by the user and then replaced by someone else. It needs to be `for_uploader_on_approve` instead.

Reply:
Only the two `own_` were wrong, the last for rejected was correct but is now wrong. Rejected replacements will never be approved and as such will never have an original uploader on approve

Reply:
I changed all 3 in bbb0b77e31745b67aba24f8535256134c4ff9f17, but fixed the rejected one in 659205d878de417bc1bfa7f3aa17577bd111b47c, so it should be correct